client/visualizations.yml

```diff
@@ -0,0 +1,30 @@
+aceeditor:
```

Comment:
The ID here does it need to match the vis-id in the XML? Maybe add a small comment on top?

Reply:
@bgruening I added a comment to the `.yml` to clarify this.